File: Hibernate_Notes_Demo/hibernate/src/main/java/com/wipro/hibernate/entity/Employee.java
package com.wipro.hibernate.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name="Emp_Table")
public class Employee {              // Entity class which associated directly with table
	
	@Id
	private int eid;
	
	@Column(name="emp_name")
	private String ename;
	
	private double salary;
	
	public Employee() {
		
		
	}

	public Employee(int eid, String ename, double salary) {
		super();
		this.eid = eid;
		this.ename = ename;
		this.salary = salary;
	}

	public int getEid() {
		return eid;
	}

	public void setEid(int eid) {
		this.eid = eid;
	}

	public String getEname() {
		return ename;
	}

	public void setEname(String ename) {
		this.ename = ename;
	}

	public double getSalary() {
		return salary;
	}

	public void setSalary(double salary) {
		this.salary = salary;
	}

	@Override
	public String toString() {
		return "Employee [eid=" + eid + ", ename=" + ename + ", salary=" + salary + "]";
	}
	
	
	
	
	
	

}

File: React_Project_Src/src_components_props_state/Employee.js
import { Component } from "react";



export class Employee  extends Component{


            constructor(props){
                        super(props);
                    

                this.state = {

                        eid:101,
                        ename: this.props.name,
                        salary: 90000


                }



                         console.log(this.props)
            console.log(this.state)


            }


                    



    render(){

            return (    

                <>
                    <h1>Welcome to Employee : {this.props.name}</h1>
                   
                  
                   <h2>Eid:  {this.state.eid}</h2>
                    <h2>Ename: {this.state.ename}</h2>
                    <h2>Salary :{this.state.salary}</h2>
                
                
                </>




            );



    }







}